tests/plugins/dut_monitor/thresholds.yml

```diff
@@ -0,0 +1,16 @@
+# Required field. Used when test run performed on setup which is not defined below.
+default:
+  cpu_total: 90
+  cpu_process: 60
+  cpu_measure_duration: 10
+  cpu_total_average: 90
+  ram_peak: 80
+  ram_delta: 1
+  hdd_used: 80
+
+x86_64-mlnx_msn2011-r0:
+  hwsku:
+    ACS-MSN2010:
```

Comment:
This structure assues that one platform may contain multiple hwsku. But generally, one platform corresponds to one hwsku, it's a 1-1 relation. Right? Then this definition has duplicated information and can be simplified. Maybe just:
```
x86_64-mlnx_msn2011-r0:
    ram_peak: 85
    cpu_total_average: 85
```
or
```
ACS-MSN2010:
    ram_peak: 85
    cpu_total_average: 85
```

Reply:
Platform may contain multiple hwsku, and also hwsku for different platforms can have same name. For example: https://github.com/Azure/sonic-buildimage/tree/master/device/mellanox/x86_64-mlnx_msn2700-r0.
For this reason thresholds.yml structure gives possibility to define several different hwsku per platform.

Logic is next:
- First it take threshold values from "default" key
- Then if platform is defined we take values from platform defaults
- If hwsku is defined for specific platform, it takes values for specific hwsku

If platform is not defined it take defaults.

Reply:
OK